Sources/Core/GTMSessionUploadFetcher.m

```diff
@@ -1513,6 +1513,11 @@ - (void)chunkFetcher:(GTMSessionFetcher *)chunkFetcher
   } else {
     // The chunk has uploaded successfully.
     int64_t newOffset = self.currentOffset + previousContentLength;
```

Comment:
Consider moving the non-uploadSizeReceived case into an `else` clause, since `self.currentOffset` involves a method call and locking. So this could be:

```
int64_t newOffset;
NSString *uploadSizeReceived = ...
if (uploadSizeReceived) {
  newOffset = [uploadSizeReceived longLongValue];
} else {
  newOffset = self.currentOffset + previousContentLength;
}
```

Reply:
Done.